joomla/joomla-cms
# [4.0] Multilang: add title when articles `showassociations` is set for frontend

Pastch similar as https://github.com/joomla/joomla-cms/pull/27714

### Summary of Changes
Multilingual site.
When the Articles Options `Associations` field is set to `Show`, image flags or Content Language Native title are displayed allowing to switch directly to the associated articles.
1. for category blog or single article in the info-block
2. Next to the title in category list.

But, similar issue as in https://github.com/joomla/joomla-cms/pull/27714 , no title is displayes when flag is not used for a specific content language.

This PR adds the title.
### Testing Instructions
Create a multilingual site.
Open one of the content languages and set the image value to none (below for German de-DE)
Set Articles: Options `Associations` field to`Show`
Make sure an article at least is associated to the article tagged to the Content Language which image is set to None.

Display single article menu item for one of these articles.
Display a blog, or featured or list of the article category concerned.

Mouse over the sef badge (DE in the example below)

### After patch
<img width="351" alt="Screen Shot 2020-01-31 at 09 21 06" src="https://user-images.githubusercontent.com/869724/73523374-0ce6c200-440b-11ea-98d2-b8b0833f4ff5.png">

<img width="377" alt="Screen Shot 2020-01-31 at 09 21 54" src="https://user-images.githubusercontent.com/869724/73523415-2556dc80-440b-11ea-863c-95fa84cb854f.png">

<img width="377" alt="Screen Shot 2020-01-31 at 09 22 23" src="https://user-images.githubusercontent.com/869724/73523460-3acc0680-440b-11ea-9d1d-b548b09b6535.png">





### Actual result



### Documentation Changes Required
